### ISSUE TITLE
Add subcommand to divide data into training set and test set

### ISSUE THREAD

**Issue by ysk24ok** [xxthermidorxx/cpmf#14]
Add subcommand (like `cv_split`) to divide data into training set and test set for cross validation.

After converting MovieLens data to cpmf format,

```sh
$ python scripts/convert_movielens.py PATH/ml-1m/ratings.dat --separator :: > input/ml-1m
```

users will split the data into training set and test set by the following command.

```sh
$ ./mf cv_split input/ml-1m --training_size 0.8
```

or

```sh
$ ./mf cv_split input/ml-1m --test_size 0.2
```
